internal/plugin/connectors/tcp/mssql/mock/mock.go

```diff
@@ -83,3 +85,16 @@ func (n *NetConn) Write([]byte) (numBytes int, err error) {
 func FakeTdsBufferCtor(r io.ReadWriteCloser) io.ReadWriteCloser {
 	return r
 }
+
+// MockTdsBufferCtor returns a Buffer (with a fake transport) that can be written & read
+func MockTdsBufferCtor(r io.ReadWriteCloser) io.ReadWriteCloser {
```

Comment:
Should drop the 'Mock' and call this 'TdsBufferCtor', otherwise some linters complain about s-s-s-stuttering, since a call would be mock.MockTdsBufferCtor. Same for 'MockTransport' below.